tests/tests-hz38/src/main/java/com/hazelcast/simulator/tests/quorum/QuorumMapTest.java

```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.simulator.tests.quorum;
+
+import org.junit.Assert;
+
+import com.hazelcast.core.IMap;
+import com.hazelcast.quorum.QuorumException;
+import com.hazelcast.simulator.test.AbstractTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+
+public class QuorumMapTest extends AbstractTest {
+
+    // properties
+    public int keyCount = 100;
+    public int gracePeriodMillis = 2000;
+
+    private volatile LastClusterSizeChange lastClusterSizeChange;
+    private IMap<Long, Long> map;
+    private int quorumCount;
+
+    @Setup
+    @SuppressWarnings("unchecked")
+    public void setup() {
+        this.lastClusterSizeChange = new LastClusterSizeChange(0L,
+                getMemberCount());
+        this.map = targetInstance.getMap(name);
+        this.quorumCount = targetInstance.getConfig()
+                .getQuorumConfig("map-quorum-ref").getSize();
+    }
+
+    private int getMemberCount() {
+        return targetInstance.getCluster().getMembers().size();
+    }
+
+    @TimeStep
+    public void testPut(BaseThreadState state) {
+        final long key = state.randomInt(keyCount);
+        final int memberCount = getMemberCount();
+        LastClusterSizeChange lastChange = lastClusterSizeChange;
+
+        if (lastChange.timestamp + gracePeriodMillis > System
```

Comment:
please do the whole if statement on a single line. The current formatting is confusing.